data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+			`	The man begins speaking to you after the woman stops. "Rather impressive work, I might say."`
+			choice
+				`	"Wait, you have a record on me?"`
+				`	"What do you need me to do?"`
+					goto mission
+	
+			`	"The Deep keeps a record on a lot of things, especially when we notice a certain individual doing a lot of humanitarian deeds."`
+			branch plot
+				has "main plot completed"
+			`	The man begins to tell you the mission.`
+				goto mission
+	
+			label plot
+			`	"Besides," the woman chimes in, "what do you expect after having been involved in the war?" The man begins to tell you the mission.`
+	
+			label mission
+			`	"We need you to find a Star Queen that recently went missing in the Far North region and return to <destination> once you have located it. Aboard this Star Queen was a number of top scientists intending to do research on the Wolf-Rayet star in Gorvi, which, due to its location so close to the northern anarchist worlds, has very little known about it."`
+			`	"We received word just yesterday that just after refueling on Prime, a large pirate fleet entered the system. The Star Queen's escorts were sadly destroyed." He seems to choke up after telling you that. "The pirates then boarded and took control of the Star Queen. We then lost all connection to the vessel, as we have no idea where it could have gone from there." He takes a moment to collect himself after telling you the situation.`
+			choice
+				`	"I would be glad to help. I'll prepare my ship immediately."`
+					accept
+				`	"Why do you seem so sad about the situation?"`
+					goto sad
+				`	"What will I receive in exchange for helping you?"`
+				`	"Why can't you locate it yourself?"`
+					goto locate
+	
+			`	"We will pay you <payment> if you are able to at least find where the Star Queen is, but there may be more reward if you are able to return it."`
+				goto end
+			
+			label sad
+			`	The man pauses before answering you, as if trying to choose his words carefully. "I personally know a lot of the scientists on board that Star Queen, and I just know that someone's going to wind up dead in this situation, and it isn't every day that someone you know gets held captive by pirates. This whole thing is just stressful for me."`
+				goto end
+	
+			label locate
+			`	"The Star Queen had heavy escorts that shouldn't have been easy to destroy, but seeing as how they were we need someone to forge ahead and find the Star Queen now while we assemble a proper rescue fleet. In a situation like this, every minute matters, and if someone like you can find the Star Queen fast then that means that there is a higher chance everyone can come out of this alive."`
```

Comment:
> but seeing as how they were we need someone...

It looked weird to me when I read this line the first time, before I realized that it was talking about how the escorts were destroyed.
Try something like
> but seeing as how they were overwhelmed we need someone...